cw2dmk.c

```diff
@@ -134,6 +134,8 @@ int accum_sectors = 0;
 int menu_err_enabled = 0;
 volatile int menu_intr_enabled = 0;
 volatile int menu_requested = 0;
+unsigned step_ms = 6;
```

Comment:
Although the use of the type `unsigned` by itself is a perfectly legal type under ISO C, so it is purely a stylistic convention, I would still prefer it stated the implicit `int`.  Prior to this commit the only other time `unsigned` stood alone was with the recent `quirk` commit adding line 140, otherwise the `unsigned` type was previously expressed as `unsigned int` in the code base.  Without the `int` my brain has to reparse reading the solo `unsigned` type twice to convince itself that it didn't miss the integer width of the type and that it's intentionally missing.  You don't have to change the definitions.  I just wanted to explain my take.